automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala

```diff
@@ -91,17 +94,16 @@ abstract class ClusterFixtureSpec extends fixture.FreeSpec with BeforeAndAfterAl
   }
 
   def getClusterRequest(): ClusterRequest = {
-    val machineConfig = Some(
+    val machineConfig =
       RuntimeConfig.DataprocConfig(
         numberOfWorkers = 0,
         masterDiskSize = 500,
-        masterMachineType = "n1-standard-8",
-        workerMachineType = Some("n1-standard-8")
+        masterMachineType = MachineTypeName("n1-standard-8"),
```

Comment:
thanks for fixing this....I thought about doing this in last big PR, but held back from it a few times :)